### PR TITLE
Wt 6664 fetch posted time

### PR DESCRIPTION
**Pull request by pfilippi24**
Added response to update status endpoint. To report update success and errors.

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -603,6 +607,25 @@ components:
...
+            description: ID of the time group to be updated.
+          fetchClientId:
+            type: string
+            description: The id of the fetch client that received this time group.
+          updateStatus:
+            type: string
+            enum:
+              - SUCCESS
+              - TIME_GROUP_NOT_FOUND
+              - UNEXPECTED_ERROR
```
Comment (vyshane) [inline spec/openapi.yaml]: Can't we use HTTP codes 200, 404 and 500 for the above?

Comment (vyshane) [reply spec/openapi.yaml]: Ah, sorry, we need to get status result for multiple time groups. In this case, wouldn't it be simpler for both API and clients to only do status updates for group at a time? We can do HTTP requests in parallel if we need to.

**Review by vyshane — CHANGES_REQUESTED** [review]
To keep API simple, let's set posted time group status for one group at a time.

File: spec/openapi.yaml (hunk 1 above)
Comment (pfilippi24) [reply spec/openapi.yaml]: That sounds cleaner. Updated

**Review by vyshane — APPROVED** [review]

**Pull request merged**